### ISSUE TITLE
update bootstrap-sass package for deprecated "/ for division"

### ISSUE THREAD

**Issue by jwhaney** [TNRIS/tnris.org#369]
after #368 updates, bootstrap-sass package needs updating. see comment below from the PR mentioned.
---
looks like my local build of your feature/security branch works. just an fyi, there is some feedback in the terminal when the webpack dev server is started. see below. this is just a note for something we might have to address in the future. i think it would be a fairly simple fix running a find & replace to resolve with the recommendation.

**EDIT:** sorry, that path is from a node_modules package so we would need to update that package. looks like it's coming from `bootstrap-sass`.

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($grid-gutter-width, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
368 │ $navbar-padding-horizontal:        floor(($grid-gutter-width / 2)) !default;
    │                                           ^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss 368:43  @import
    scss/partials/_variables.scss 1:9                                                @import
    stdin 1:9                                                                        root stylesheet

DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($grid-gutter-width, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
368 │ $navbar-padding-horizontal:        floor(($grid-gutter-width / 2)) !default;
    │                                           ^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss 368:43  @import
    scss/partials/_variables.scss 1:9                                                @import
    scss/partials/_2014-forum.scss 2:9                                               @import
    stdin 2:9                                                                        root stylesheet

DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($navbar-height - $line-height-computed, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
369 │ $navbar-padding-vertical:          (($navbar-height - $line-height-computed) / 2) !default;
    │                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss 369:37  @import
    scss/partials/_variables.scss 1:9                                                @import
    stdin 1:9                                                                        root stylesheet

DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($navbar-height - $line-height-computed, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
369 │ $navbar-padding-vertical:          (($navbar-height - $line-height-computed) / 2) !default;
    │                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss 369:37  @import
    scss/partials/_variables.scss 1:9                                                @import
    scss/partials/_2014-forum.scss 2:9                                               @import
    stdin 2:9                                                                        root stylesheet

```

**Comment by jwhaney** [comment]
https://github.com/TNRIS/tnris.org/pull/368#issuecomment-869897263

**Comment by jwhaney** [comment]
closed via #370
